builtin/provisioners/chef/ssh_provisioner.go

```diff
@@ -0,0 +1,74 @@
+package chef
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/communicator"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const (
+	installURL = "https://www.chef.io/chef/install.sh"
```

Comment:
Can we get this changed to be a default parameter that can be overridden? Not everyone uses chef in a public internet configuration and HTTP proxies are not always available.